README.md

```diff
@@ -2,68 +2,54 @@
+All reader test cases are stored in the directory `out/reader_tests`. They follow the directory structure:
 
 ```
-source "$( poetry env list --full-path | grep Activated | cut -d' ' -f1 )/bin/activate"
+|-- {table_name}
+  |-- case_metadata.json
+  |-- delta
+  | |-- _delta_log
+  |   |-- ...
+  | |-- part-0000-dsfsadf-adsfsdaf-asdf.snappy.parquet
+  | |-- ...
+  |-- expected
+    |-- latest
+      |-- expected_metadata.json
+      |-- table_content.parquet
+    |-- v1
+      |-- expected_metadata.json
+      |-- table_content.parquet
 ```
 
-Run `python -m dat.main` to make sure everything is working properly.  You should see instructions on how to use DAT, including the commands you can run.
+Each test case is a folder, named for it's test. It contains:
 
-## How to build tables
+ * `case_metadata.json`: document that provides the names and human-friendly description of the test.
+ * `delta`: the root directory of the Delta Lake table.
+ * `expected`: a folder containing expected results, potentially for multiple versions of the table. At a minimum, there is a folder called `latest` containing the expected data for the current version of the table. There may be other folders such as `v1`, `v2`, and so on for other versions for testing time travel. 
 
-Create the reference tables with `python -m dat.main write-generated-reference-tables`.
+To test a reader, readers should first identify the test cases:
 
-This will create reference tables as follows:
+ 1. List the `out/reader_tests` directory to identify the root of each delta table.
+ 2. List each subdirectories `expected` directory; each of these folders represents a test case.
 
-```
-out/
-  tables/
-    generated/
-      reference_table_1/
-        delta/
-          _delta_log/
-            000.json
-            001.json
-          parquet_files
-        parquet/
-          table_content.parquet
-        table_metadata.json
-      reference_table_2/
-        ...
-```
-
-Let's take a look at the contents of `reference_table_1`:
-
-* `delta`: This is a Delta Lake table.  It consists of a transaction log (`_delta_log`) and data stored in Parquet files.
-* `parquet/table_content.parquet`: Contains all the data that should be in the Delta Lake table when it's read.  This is the expected value.
-* `table_metadata.json`: contains the metadata that the `_delta_log` is expected to contain.  This is another expected value.
+Then for each test case:
 
-For each reference table, you should run these checks:
+ 1. Load the corresponding version of the delta table
+ 2. Verify the metadata read from the delta table matches that in the expected_metadata.json
+ 3. Attempt to read the delta tables data:
+   a. If the delta table uses an unsupported version (as determined from expected_metadata.json), verify an appropriate error is returned.
```

Comment:
`uses a version unsupported by the reader connector`

i.e. clarify what "unsupported" means.